docs/faq.md

```diff
@@ -21,7 +21,7 @@ to looking for pypi packages signed with MD5 signatures. If you've enabled FIPS
 (or are using a build that has FIPS pre-enabled), MD5 is disabled in the kernel
 (due to being a weak hashing-method). You can either disable FIPS (not
 recommended) or explicitly force `pip` to use a different signature-index. The
-latter is detailed in the Linux section of the :doc:`usage <usage>` document.
+latter is detailed in the Linux section of the [usage] (usage.html) document.
```

Comment:
Oops, got a space in there between the bracket and paren: `] (`

Open a new pr to fix?